test/firebase_storage_mocks_test.dart

```diff
@@ -40,10 +40,10 @@ void main() {
   });
 }
 
-Future<String> getGsLink(StorageReference storageRef) async {
+Future<String> getGsLink(Reference storageRef) async {
   return Uri(
     scheme: 'gs',
-    host: await storageRef.getBucket(),
-    path: storageRef.path,
+    host: storageRef.bucket,
```

Comment:
As I wrote above, you'll have to update the getters in `MockStorageReference` for this code to work properly.